tests/backends/cursors/mysql.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
IMO you should move `LoggingCursor`, `LoggingCursorFactory` inside backend specific test (`tests/backends/tests.py` or separate `tests/backends/test_cursors.py`) and create base mixins for `LoggingCursorMixin`, `LoggingCursorFactoryMixin` e.g.:
```python
class LoggingCursorMixin(object):
    def __init__(self, *args, **kwargs):
        bucket = kwargs.pop('bucket')
        super(LoggingCursorMixin, self).__init__(*args, **kwargs)
        self.bucket = bucket

    def execute(self, sql, *args, **kwargs):
        self.bucket.append(sql)
        super(LoggingCursorMixin, self).execute(sql, *args, **kwargs)

...

@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL specific test.')
class MySQLCursorOptionsTestCase(TestCase):
    class MySQLLoggingCursor(LoggingCursorMixin, Cursor):
        pass

...

@unittest.skipUnless(connection.vendor == 'sqlite', 'SQLite specific test.')
class SQLiteCursorOptionsTestCase(TestCase):
    class SQLiteLoggingCursor(LoggingCursorMixin, SQLiteCursorWrapper):
        pass

...
```
etc.